### ISSUE TITLE
iconWithIdentifier with swift

### ISSUE THREAD

**Issue by jpodcedensek** [PrideChung/FontAwesomeKit#84]
Hi, is there a way to use iconWithIdentifier in Swift code?

**Comment by hebbian** [comment]
No, there isn't last time I checked on latest installation from pod

**Comment by jpodcedensek** [comment]
I solved that in a dirty way with
 `
        let iconIdentifier = "iconIdentifier"
        if iconIdentifier.containsString("fa-"){
            let iconCodes = FAKFontAwesome.allIcons() as NSDictionary
            
            if iconCodes.allKeysForObject(iconIdentifier.stringByReplacingOccurrencesOfString("fa-", withString: "")).count > 0{
                if let iconCode = iconCodes.allKeysForObject(iconIdentifier.stringByReplacingOccurrencesOfString("fa-", withString: ""))[0] as? String{
                    iconLabel.attributedText = FAKFontAwesome(code: iconCode, size: 40).attributedString()
                }
            } else {
                print("Can't find FontAwesome icon for identifier ", iconIdentifier)
            }
            
        }`
